tools/release-ci/pkg/config/batch.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import "fmt"
+
+const (
+	PullRequestPrefix = "pr-logs"
+	PullRequestDirectory = PullRequestPrefix + "/directory"
+)
+
+// Repo contains configuration for a published
+// repository version involved in a job
+type Repo struct {
+	// RepoOwner is the GitHub organization
+	// that triggered this build, provided as
+	// $REPO_OWNER by Prow.
```

Comment:
REPO_OWNER and REPO_NAME are only added in pods. Do we want to add them in Jenkins jobs?